tests/atomistics/structure/test_atoms.py

```diff
@@ -422,11 +422,38 @@ def test_get_bonds(self):
         # print H2O.get_bonds(radius=2.)[0]
         # print np.sum(H2O.get_masses())/H2O.get_volume()
 
-    # def test_get_symmetry(self):
-    #     cell = 2.2 * np.identity(3)
-    #     Al_sc = Atoms('AlAl', positions=[(0, 0, 0), (0.5, 0.5, 0.5)], cell=cell)
-    #     Al_sc.repeat([2, 2, 2])
-    #     self.assertEqual(len(Al_sc.get_symmetry()['translations']), 768)
+    def test_get_symmetry(self):
+        cell = 2.2 * np.identity(3)
+        Al = Atoms('AlAl', positions=[(0, 0, 0), (0.5, 0.5, 0.5)], cell=cell).repeat(2)
+        self.assertEqual(len(set(Al.get_symmetry()['equivalent_atoms'])), 1)
+        self.assertEqual(len(Al.get_symmetry()['translations']), 96)
+        self.assertEqual(len(Al.get_symmetry()['translations']), len(Al.get_symmetry()['rotations']))
+
+    def get_equivalent_voronoi_vertices(self):
```

Comment:
The two test functions get_equivalent_voronoi_vertices(self) L 432 and get_equivalent_voronoi_vertices(self) L448 seem to be the same.